scout/server/blueprints/variants/views.py

```diff
@@ -293,7 +296,14 @@ def cancer_variants(institute_id, case_name):
     else:
         page = int(request.args.get("page", 1))
         form = CancerFiltersForm(request.args)
-        form.chrom.data = request.args.get("chrom", None)
+        # set chromosome to all chromosomes
+        form.chrom.data = request.args.get("chrom", "")
+        # set gene panels to case-specific gene panels
```

Comment:
Same here!

Reply:
(And in general that implies we have a bit too much code duplication between the different variant type views, but that is for another PR!)